build/localnet/cluster-manager/utils.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
+	v1 "k8s.io/api/core/v1"
```

Comment:
One of the following:

1. Remove `v1`
2. s/v1/k8s

Reply:
1. 

https://user-images.githubusercontent.com/29378614/219801348-6832323e-0171-43f5-bf7f-0e546f451761.mp4

VSCode won't allow that for some reason.

2. Ok